jekyll/_cci2/faq.md

```diff
@@ -165,6 +165,21 @@ No.
 ### Can I build fork PR’s using Workflows?
 We do not support fork PR’s yet.
 
+### Can workflows be scheduled to run at a specific time of day?
+Yes. For example, to run a workflow at 4 PM use `"0 16 * * *"` as the value for the `cron:` key. Times are interpreted in the UTC time zone.
+
+### What time zone is used for schedules?
+Coordinated Universal Time (UTC) is the time zone in which schedules are interpreted.
+
+### Why didn’t my scheduled build run? 
+You must specify exactly the branches on which the scheduled workflow will run.
```

Comment:
Agreed - this will need some clear explanation, since the schedule needs to be correct _and_ present on the HEAD of the branch that is to be built, afaict.

Reply:
ok, thanks, I added this to the FAQ as well: "You must specify exactly the branches on which the scheduled workflow will run and push that `config.yml` to the branch you want to build. A push on the `master` branch will only schedule a workflow for the `master` branch."